view/frontend/web/js/slide-widget-mixin.js

```diff
@@ -0,0 +1,87 @@
+define([
+    'jquery',
+    'underscore',
+    'Magento_PageBuilder/js/widget/show-on-hover',
+    'Magento_PageBuilder/js/widget/video-background',
+    'CustomGento_Cookiebot/js/video-blocker-widget'
+], function ($, _, showOnHover, videoBackground, createVideoBlocker) {
+    'use strict';
+
+    return function (originalWidget) {
+        return function (config, element) {
+            let videoElement = element[0].querySelector('[data-background-type=video]');
+            let PreviousStatus = '';
+            if (!videoElement) {
+                return;
+            }
+
+            let viewportElement = document.createElement('div');
+            let $slider = videoElement.getAttribute('data-video-src')
+            const blockVideoConsentConfig = window.cookiebotConfig && window.cookiebotConfig.blockVideosUntilConsent;
```

Comment:
Shouldn't we check this directly in the beginning and if the option is not enabled, just return / call the original widget?